components/PackageOverview.js

```diff
@@ -69,74 +109,146 @@ export const PackageOverview = () => {
     >
       ${Object.keys(versions).map(VersionOption)}</select
     >
-    <${Directory}
-      packageName=${name}
-      rootMeta=${directory}
-      version=${version}
-      filter=${fileSearchTerm}
-    />
+    ${fileSearchTerm
+      ? html`
+          <div>
+            ${flatFiles
+              .filter(file =>
+                file.path
+                  .toLowerCase()
+                  .match(
+                    fileSearchTerm
+                      .toLowerCase()
+                      .replace(/([.?*+^$[\]\\(){}|-])/g, '\\$1')
```

Comment:
What is this?

Reply:
It's a little regex to remove unescaped regex characters from the string as `string.match` applies a regex of the inputted string. I.e. if I type `asd[` into the search box, it would error as the `[` would register as an unclosed regex set, this just replaces those characters with their escaped versions to stop it erroring.